src/SubjectPreview.js

```diff
@@ -1,44 +1,164 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import {makeStyles} from "@material-ui/core/styles";
+import ListItem from "@material-ui/core/ListItem";
 import ListItemText from "@material-ui/core/ListItemText";
-import SubjectDialog from "./SubjectDialog";
+import Box from "@material-ui/core/Box";
+import Grid from "@material-ui/core/Grid";
 import Typography from "@material-ui/core/Typography";
-import ListItem from "@material-ui/core/ListItem";
+
+import SubjectDialog from "./SubjectDialog";
+import QuantityIndicator from "./QuantityIndicator";
 import {daysInWeek} from "./constants";
 
+
 const formatDay = (day) => {
```

Comment:
Otázka je, jestli tohle není zbytečné. Přidal jsem to tam asi já, takže tenhle comment je takovej self burn :smile:

Reply:
Je no :D